.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just to get my linter to stop complaining, will put in the the real properly configured environment files in another PR.